client/package.json

```diff
@@ -425,6 +425,24 @@
         "title": "BitBake: Parse all recipes",
         "description": "This command runs bitbake in parse-only mode."
       },
+      {
+        "command": "bitbake.start-toaster-webview",
+        "title": "BitBake: Start Toaster Webview",
+        "description": "Start the Toaster web interface in a new window or reveal the existing one.",
+        "icon": "$(play-circle)"
+      },
+      {
+        "command": "bitbake.start-toaster-in-browser",
```

Comment:
We should probably only keep one of browser/webview.

Reply:
So which one do we keep?

Reply:
IMO, we should go for the browser which saves the potential work for the toaster team to adapt our changes.